README.md

```diff
@@ -42,69 +70,256 @@ If you submit a pull request to implement a new feature without going
 through the RFC process, it may be closed with a polite request to
 submit an RFC first.
 
-## Gathering feedback before submitting
+## Gathering feedback before submitting an RFC
+[Gathering feedback before submitting an RFC]: #gathering-feedback-before-submitting-an-rfc
 
 It's often helpful to get feedback on your concept before diving into the
 level of API design detail required for an RFC. **You may open an
 issue on this repo to start a high-level discussion**, with the goal of
 eventually formulating an RFC pull request with the specific implementation
- design. We also highly recommend sharing drafts of RFCs in `#dev-rfc` on 
+ design. We also **_highly recommend_** sharing drafts of RFCs in `#dev-rfc` on
 the [Ember Discord](https://discord.gg/emberjs) for early feedback.
 
-## The process
+## The Process
+[The Process]: #the-process
 
-In short, to get a major feature added to Ember, one must first get the
-RFC merged into the RFC repo as a markdown file. At that point the RFC
-is 'active' and may be implemented with the goal of eventual inclusion
-into Ember.
+The process for a successful RFC involves several [Stages] that take
+the proposed change all the way from proposal to implemented and widely adopted.
+A newly opened RFC is in the first stage, appropriately named, [Proposed].
+
+### How to create a new RFC
+[How to create a new RFC]: #how-to-create-a-new-rfc
 
 * Fork the RFC repo http://github.com/emberjs/rfcs
-* Copy the appropriate template. For most RFCs, this is `0000-template.md`, 
+* Copy the appropriate template. For most RFCs, this is `0000-template.md`,
 for deprecation RFCs it is `deprecation-template.md`.
 Copy the template file to `text/0000-my-feature.md`, where
 'my-feature' is descriptive. Don't assign an RFC number yet.
 * Fill in the RFC. Put care into the details: **RFCs that do not
 present convincing motivation, demonstrate understanding of the
 impact of the design, or are disingenuous about the drawbacks or
 alternatives tend to be poorly-received**.
-* Fill in the relevant core teams. Use the table below to map from projects to 
+* Fill in the relevant core teams. Use the [table below](#relevant-teams) to map from projects to
 teams.
 * Submit a pull request. As a pull request the RFC will receive design
 feedback from the larger community, and the author should be prepared
-to revise it in response.
-* Find a champion on the relevant core team. The champion is responsible for 
-shepherding the RFC through the RFC process and representing it in core team 
+to revise it in response. The RFC is now in the [Proposed] stage.
+* Find a champion on the relevant core team. The champion is responsible for
+shepherding the RFC through the RFC process and representing it in core team
 meetings.
-* Update the pull request to add the number of the PR to the filename and 
+* Update the pull request to add the number of the PR to the filename and
```

Comment:
Do we still want this? This number is going to grow a lot more quickly if we have PRs to update RFCs.

Reply:
Would we stop numbering RFCs then? 

It shouldn't be any more onerous for the numbers growing quickly, but it is annoying that it needs to be added after opening the PR. RFCs keep the # from their initial PR to 'Accepted'

Reply:
I don’t know whether we should stop numbering or not, but there’s an already incorrect impression that RFC-800 means that there were 800 RFCs that were at least proposed by that point. This only gets worse with more actual PRs now. We could just pick the number sequentially when we actually merge. Possibly could even do this with automation. Not a big deal either way and not a blocker.